tests/abstract_test_ods_output_diff.py

```diff
@@ -65,7 +66,7 @@ def _generate(
         input_path: Path = INPUT_PATH,
         from_date: date = MIN_DATE,
         to_date: date = MAX_DATE,
-        generation_language: str = "en",
+        generation_language: Optional[str] = None,
```

Comment:
I suggest leaving it as it was, because the None case doesn't seem to add value: could you clarify the rationale for this (I want to make sure I'm not missing something).

Reply:
As it is coded now, if you pass 'en' as the generation language 'abstract_test_ods_output_diff.py' sees it as default and falls back 'jp' reports to the default language set in its country profile which is JP. This makes it impossible to generate an 'en' report for a report that has non-'en' default. 

Basically, if you set the generation language to 'en' the report will fall back to its default, which, to me, seems like unexpected behavior. I would think passing 'None' would cause the report to fall back to the default.

The other way to resolve this is to have no defaults or have 'en' as the default for all reports. Again, I don't see that as expected behavior.

Reply:
Oh, I see, this is probably a small problem in line 84 and 85: `-g` should be passed regardless. Anyway, thanks for the explanation. I'm now OK with either of:
* what you did or
* pass `-g` at all times on line 85